arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
Yes but we introduced one more "state" property in our ARM response to be used by service other than ARM
The ASAzure side already checked-in related change. There were some review process in service side so that we checked in the change

To be clear, "provisioningState" is still there, but we do not want it get used by SDK.